atat_provisioning_wizard_api.yaml

```diff
@@ -551,8 +551,16 @@ components:
         - $ref: '#/components/schemas/BaseObject'
         - type: object
           properties:
+            name:
```

Comment:
What does this name represent?

Reply:
This name represents the portfolio draft defined in the `portfolio_step`. Do you think it should be named as `portfolioDraftName` or `portfolioName` to make it clear?

Reply:
@juliusfitzhugh-ccpo No I think name is in keeping with what we've done in other schema objects.  If we're going to deviate from that, let's discuss changing them all to be more specific/for clarity.